### PR TITLE
Add missing filters to litigations

### PR DESCRIPTION
**Pull request by aabdaab**
* Add more filters toggle button to the litigation cases search index
* New filters: filter by `jurisdiction`, `party_type` and `side types` (types and names) from LitigationSide

8DB23F99-4E20-428C-88A4-71E18BC9DA37

image

Screenshot 2020-01-14 at 14 05 16

### CODE REVIEW

**Comment by aabdaab** [comment]
actually, gonna mark it as `WIP`, I just noticed they added more filters to the spreadsheet

**Comment by aabdaab** [comment]
ready to review ☝️

**Comment by aabdaab** [comment]
I added the sorting, gonna merge it now.

File: app/services/queries/cclow/litigation_query.rb [hunk 1]
```diff
@@ -74,11 +78,132 @@ def filter_by_to_status
         scope.joins(:events).where(events: {id: event_ids, event_type: params[:status]}).distinct
       end
 
+      # rubocop:disable Metrics/CyclomaticComplexity
+      def filter_by_litigation_sides
```
Comment (tsubik) [inline app/services/queries/cclow/litigation_query.rb]: Crap we don't have any tests for queries? This seems complicated now :/

**Pull request merged**